ceremonies/src/lib.rs

```diff
@@ -1754,6 +1772,8 @@ impl<T: Config> Pallet<T> {
 					participant,
 					Reputation::VerifiedUnlinked,
 				);
+				<ReputationCount<T>>::mutate((&cid, cindex), |b| *b += 1); // safe, as reputation_count is limited by the number of locations available on earth
```

Comment:
is there a reason why you mutate every single issuance instead of counding issuances in your loop and mutating once? db access is expensive and should be minimized

Reply:
yes, makes total sense👍🏽